src/citation_vs_reviewscore.py

```diff
@@ -0,0 +1,15 @@
+import csv
+import os
+import sys
+
+src_path = os.getcwd()
+data_path = src_path.strip('src') + 'data/'
+input_file = 'acta_a_citations_review_scores_utf8.csv'
+file_path = data_path + input_file
```

Comment:
compose this using Path when you switch over and not string operators.  more robust and better